src/components/CoursesTabs.js

```diff
@@ -167,9 +218,9 @@ export default function CoursesTabs() {
         <TabContainer>
           {courses_fetched && !courses_error && courses.length !== 0 && (
             <>
-              {off_campus && <OffCampus classes={classes} />}
-              {multiple_meetings && <MultipleMeetings classes={classes} />}
-              {<UpdateInfo classes={classes}/>}
+              {campus_visible && off_campus && <OffCampus classes={classes} set_campus_visible={set_campus_visible} />}
+              {meetings_visible && multiple_meetings && <MultipleMeetings classes={classes} set_meetings_visible={set_meetings_visible} />}
+              {update_info_visible && <UpdateInfo classes={classes} set_update_info_visible={set_update_info_visible}/>}
```

Comment:
This looks pretty good and the ability to dismiss is a neat feature. I don't know how users will feel that every time they refresh the page, the formerly dismissed alert will be back though. So at most they could enjoy not seeing that little information box for 30 minutes at a time lol

Also, we might want to think about generalizing this information card component and just calling that generalized one with a provided title, description, and function. Could be a later feature tho

Reply:
Yeah, I agree that the future version should not show dismissed cards as often and It would be nice to combine all of these into one common card for sure.

Reply:
I added two issues for this #30 and #31